### ISSUE TITLE
Feed pre-trained embeddings to NVTabular

### ISSUE THREAD

**Issue by MelissaKR** [NVIDIA-Merlin/dataloader#124]
**What is your question?**
I have a dataset that includes a column feature of pre-trained embeddings. I couldn't find any documentations or examples on how this column should be passed to NVTabular. Is it treated as a continuous feature?

**Comment by MelissaKR** [comment]
@rnyak Thank you for getting back to me on this! I have a main model and in that model, let's say I have a feature for different movies. I can pass it as a regular categorical feature to then be fed to the embedding layer. My model uses PyTorch, by the way. But I have trained a different model that uses collaborative filtering which learns embeddings for these movies much better. So now, for each movie in the training and validation set for the main model, I have vectors of size n that are the learned embeddings. And I don't need to use this movie feature anymore and pass it to an embedding layer. Instead, I want to remove it from my dataset and use the learned embeddings from the second model, but I want to see if there is a straightforward way of doing this, instead of manually defining n new numeric features for each element in the new movie embeddings and pass them to NVTabular. In other words, how can I pass pre-trained embeddings as is to my model?
I hope I could clarify my question and use case.

**Comment by rnyak** [comment]
@MelissaKR thanks for the clarification. we are currently working on that and we will be creating an example shortly. Example might not be on PyT but you can adapt it to your framework I believe :) Can you please tell me what's the architecture of your `main model`? it is an MLP model? or a more complicated architecture? Besides can you share a simple screenshot what would your data look like ? contains nested 3D arrays? or is it something like below?

movie_id  movie_embedding
1                [float1, float2, ..., float64] 
2               [float1, float2, ..., float64] 
..
n               [float1, float2, ..., float64] 

or more like this
movie_id.     movie_genres_id.        movie_genres_embeddings
1                   [1, 2, 3]                          [[float1, float2, ..., float64] , [float1, float2, ..., float64] , ...]
2                   [3,5]                              [float1, float2, ..., float64] , [float1, float2, ..., float64] ]
